GoogleAnalyticsTracker.Core/TrackerBase.async.cs

```diff
@@ -71,25 +71,39 @@ private static object GetLowerCaseValueFromEnum(PropertyInfo propertyInfo, IGene
                 : value.ToString().ToLowerInvariant();
         }
 
-        private void SetRequired(IGeneralParameters parameters)
+        /// <summary>
+        /// Set properties in parameters if they are empty: TrackingId, ClientId, CacheBuster.
+        /// The CacheBuster is set when UseHttpGet flag is set.
+        /// <para>
+        /// Override to change other properties.</para>
+        /// </summary>
+        /// <param name="parameters">GA request parameters.</param>
+        protected virtual void AmendParameters(IGeneralParameters parameters)
         {
-            parameters.TrackingId = TrackingAccount;
+            if (string.IsNullOrEmpty(parameters.TrackingId))
+            {
+                parameters.TrackingId = TrackingAccount;
+            }
 
-            if (string.IsNullOrWhiteSpace(parameters.ClientId))
+            if (string.IsNullOrEmpty(parameters.ClientId))
             {
                 parameters.ClientId = AnalyticsSession.GenerateSessionId();
             }
 
-            if (string.IsNullOrWhiteSpace(parameters.UserLanguage))
```

Comment:
What happened to this one?

Reply:
`UserLanguage` is going to be removed in #126. I have mixed the code by mistake. So, this PR either should return back this code or remove it completely.

Reply:
The UserLanguage is optional, should not be set by the core `TrackerBase` class. It's related to the #126 bot it's not mentioned there.